### PR TITLE
Maksymc/number input overview

### PR DESCRIPTION
**Pull request by maksymc**
Latest updates to the number-input spec / README. 
Scope:

1. Changes related to overview ticker from Nadav #353 
~ added suffix / prefiz props
~ fix onChange signature
~ remove RTL prop
2. Updated docs to match the latest template
3. Updated screenshots

What is NOT done:
1. Code samples 
2. Style samples 
This will be covered later.

### CODE REVIEW

File: docs/number-input/README.md [hunk 1]
```diff
@@ -1,40 +1,34 @@
...
+![elements](./assets/elements.png)
 
+## Component API
 
+#### Props
 
-#### Accepted Children
+| name         | type                             | defaultValue | isRequired | description                              |
+| ------------ | -------------------------------- | ------------ | ---------- | ---------------------------------------- |
+| value        | number                           |              | yes        | Used to set & change the value of the input. If you bind this to a state in your parent component, you should also set the onChange handler, in order for the component to work. |
```
Comment (LeonidLevi) [inline docs/number-input/README.md]: Value isn't required. If value isn't supplied, component behaves uncontrolled

File: docs/number-input/README.md [hunk 2]
```diff
@@ -1,40 +1,34 @@
...
+| max          | number                           | 100          |            |                                          |
+| step         | number                           | 1            |            |                                          |
+| required     | bool                             | false        |            | Whether or not filling the value is required in a form. |
+| disabled     | bool                             | false        |            | If true, the component will not be interactable. |
+| label        | string                           |              |            | Text to display in accessibility mode.   |
+| name         | string                           |              |            | The name of the component. Behaves like the name attribute of an input element. |
+| prefix       | node                             |              |            | Inserts a component at the start of the input. |
+| suffix       | node                             |              |            | Inserts a component at the end of the input. |
+| onChange     | `(event: {value: number}): void` |              | yes        | Callback function that is fired on component blur. |
```
Comment (LeonidLevi) [inline docs/number-input/README.md]: onChange isn't required

File: docs/number-input/README.md [hunk 3]
```diff
@@ -1,40 +1,34 @@
...
+| error        | bool                             | false        |            | Sets the `:error` CSS state on the `<NumberInput>`. |
 
-This component accepts children with the following `data-slot` attribute values, in order to be displayed in specific places in its layout
 
-| data-slot | description                              | example                                  |
-| --------- | ---------------------------------------- | ---------------------------------------- |
-| prefix    | Allows you to insert a component (or components) at the start of the input | `<div data-slot="prefix">hello world</div>` |
-| suffix    | Allows you to insert a component (or components) at the end of the input. Default data slot (not specifying anything will land the children here) | `<div data-slot="suffix">hello world</div>` |
 
 ### Code Examples
```
Comment (LeonidLevi) [inline docs/number-input/README.md]: Please fix code example with prefix and suffix (it shouldn't be with nodes anymore, but with props)

File: docs/number-input/number-input-spec.md [hunk 4]
```diff
@@ -22,48 +18,66 @@ typora-copy-images-to: ../NumberInput
...
 
+Number Input improves upon the native `<input type="number">` by providing the ability to customise stepper arrows design, a common React+Typescript API, and working out the kinks of native implementations.
 
 ## Elements
 
-![elements](./elements.png)
+![elements](./assets/elements.png)
 
-The NumberInput consists of a native input and a Stepper subcomponent, which provides customisable Up and Down buttons. Also, you can add elements into it in the prefix and suffix slots, by passing the component children with the corresponding data-slot attribute.
+The **Number Input** functions as native `<input type="time">` component with additional ability to customise stepper and stepper arrows (using `::stepper`, `::stepper::up` ,`::stepper::down` subcomponents). Also number input can be customised by passing node elements into `prefix` and `suffix` properties.
```
Comment (LeonidLevi) [inline docs/number-input/number-input-spec.md]: input="time" - seems like a mistake has crept in :)

File: docs/number-input/README.md (hunk 1 above)
Comment (maksymc) [reply docs/number-input/README.md]: Done

File: docs/number-input/number-input-spec.md (hunk 4 above)
Comment (maksymc) [reply docs/number-input/number-input-spec.md]: Holy cow! Yep, my bad. Fixed.

File: docs/number-input/README.md (hunk 2 above)
Comment (maksymc) [reply docs/number-input/README.md]: Fixed

File: docs/number-input/README.md (hunk 3 above)
Comment (maksymc) [reply docs/number-input/README.md]: Done

**Pull request merged**